setup_jdk21_and_mvn.sh

```
@@ -1,51 +1,93 @@
+else
+  echo "[INFO] SDKMAN already installed."
 fi
+echo "[INFO] SDKMAN install check complete."
+
 
 
 # Load SDKMAN into the current shell session (try $HOME, then /usr/local)
+echo "[INFO] Loading SDKMAN into current shell session..."
 if [ -s "$HOME/.sdkman/bin/sdkman-init.sh" ]; then
+  export SDKMAN_DIR="$HOME/.sdkman"
   source "$HOME/.sdkman/bin/sdkman-init.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./.sdkman/bin/sdkman-init.sh was not specified as input (see shellcheck -x).

(SC1091)